rust/pact_ffi/Cargo.toml

```diff
@@ -33,7 +33,7 @@ onig = { version = "6.5.1", default-features = false }
 # FFI Inmemory sink uses the global static `LOG_BUFFER` in the pact_matching crate. This is required to be the same
 # version as the mock server crate
 pact_matching = { version = "1.2.12" }
-pact_mock_server = { version = "~1.2.16" }
+pact_mock_server = "2.1.3"
```

Comment:
Set the version to `~2.1.3` please, so it will only automatically update patch versions

Reply:
Cargo will automatically upgrade patch versions without the `~`, under the [default requirements syntax](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#default-requirements).

Reply:
> Tilde requirements specify a minimal version with some ability to update. If you specify a major, minor, and patch version or only a major and minor version, only patch-level changes are allowed.

From that link

Reply:
All good, I missed the 'only'